### PR TITLE
User can now use markdown for wikis

### PR DESCRIPTION
**Pull request by blafri**
Hey Felix,
I completed the markdown assignment. Let me know if it looks ok and if anything can be improved.

Thanks again,
Blayne

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -49,4 +49,32 @@ def is_link_active(link_path)
...
+  # Public: This methods users the redcarpet gem to parse markdown text to HTML
+  #
+  # markdown_text - String containg the markdown do be converted to html
+  #
+  # Examples
+  #
+  #   markdown_to_html('# This is a title')
+  #   # => <h1>This is a title</h1>
+  #
+  # Returns a String containg the html generated from the markdown text
```
Comment (felixclack) [inline app/helpers/application_helper.rb]: Nice comment. There's a typo here 'containing'.

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -49,4 +49,32 @@ def is_link_active(link_path)
...
+    
+    redcarpet = Redcarpet::Markdown.new(renderer,
+      tables: true,
+      autolink: true,
+      strikethrough: true,
+      superscript: true,
+      underline: true,
+      highlight: true)
+    
+    (redcarpet.render markdown_text).html_safe
```
Comment (felixclack) [inline app/helpers/application_helper.rb]: I would have used the `()` to wrap the arguments only, eg. `redcard.render(markdown_text).html_safe`.

**Comment by felixclack** [comment]
Looks good man.

**Comment by blafri** [comment]
Thanks for the help Felix.
Will merge the commit.

**Pull request merged**